### PR TITLE
Add Support for Transforming Decimal Trailing Zeroes to Exponent

### PR DESCRIPTION
**Pull request by Ang-dot**
Actual frontend testing in local

image<br>

Added updated test cases and passed all tests.
```
yarn run v1.22.19
$ jest
 PASS  src/test.js
  ✓ isCrypto (3ms)
  is crypto
    raw = true
      ✓ returns precision of 8 (13ms)
    raw = false
      ✓ returns formatted (3ms)
    noDecimal = true
      ✓ returns without decimal (1ms)
      ✓ returns decimal when less than 1 (1ms)
    abbreviated = true
      ✓ returns abbreviated format (EN) (1ms)
      ✓ returns abbreviated format (JA) (1ms)
  is fiat
    raw = true
      ✓ returns formatted raw (1ms)
    raw = false
      ✓ returns formatted with symbol (1ms)
    no decimal threshold
      ✓ returns decimal for amounts <= 100,000
      ✓ returns no decimal for amounts > 100,000 (1ms)
    noDecimal = true
      ✓ returns without decimal
      ✓ returns decimal when less than 1
    abbreviated = true
      ✓ returns abbreviated format (EN) (2ms)
      ✓ returns abbreviated format (JA) (1ms)
  Intl.NumberFormat not supported
    is BTC or ETH
      raw = true
        ✓ returns precision of 8
      raw = false
        ✓ returns currency with ISO Code (1ms)
    is fiat
      raw = true
        ✓ returns formatted raw (1ms)
      raw = false
        ✓ returns formatted with symbol (1ms)
  Format Currency correctly
    ✓ formats JPY correctly (1ms)
  Accepts object parameter
    ✓ defaults to noDecimal = false
    ✓ formats decimal places correctly (1ms)
    ✓ formats significant figures correctly (1ms)
    ✓ formats trailing decimal zeroes correctly (2ms)
    ✓ formats decimal places, significant figures and trailing decimal zeroes correctly (2ms)
    ✓ raw = true

Test Suites: 1 passed, 1 total
Tests:       26 passed, 26 total
Snapshots:   0 total
Time:        1.635s, estimated 2s
Ran all test suites.
✨  Done in 3.71s.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coingecko/cryptoformat",
-  "version": "0.7.0",
+  "version": "0.7.1",
```
Comment (ervinchai) [inline package.json:3]: Use semantic versioning!

**Comment by ervinchai** [comment]
Looks fine to me. Have we tested this against actual usages within the codebase? If yes, please attach screenshots!

File: package.json (hunk 1 above)
Comment (Ang-dot) [reply package.json]: Thanks! Comment addressed!

**Comment by ervinchai** [comment]
This looks good to me. Will hold off on merging first until we are sure that there are no edge cases. Will run this against develop branch! Thanks for the work, I'll take over from here!

**Review by ervinchai — APPROVED** [review]
LGTM!

**Pull request merged**